packages/bot-web-ui/src/components/dashboard/bot-builder/toolbox/search-box/search-icon.tsx

```diff
@@ -1,16 +1,31 @@
 import React from 'react';
+import debounce from 'lodash.debounce';
+import { RudderStack } from '@deriv/analytics';
 import { Icon } from '@deriv/components';
+import { observer } from '@deriv/stores';
 
 type TSearchIcon = {
     search: string;
     is_search_loading: boolean;
     onClick: () => void;
 };
 
-const SearchIcon = ({ search, is_search_loading, onClick }: TSearchIcon) => {
+const SearchIcon = observer(({ search, is_search_loading, onClick }: TSearchIcon) => {
+    React.useEffect(() => {
+        //this is to check after keup on bot-builder page form sent even to rudderstack
+        debounce(() => {
+            if (search && !is_search_loading) {
+                RudderStack.track('ce_bot_builder_form', {
+                    search_string: search,
+                    action: 'search',
+                });
```

Comment:
![image](https://github.com/binary-com/deriv-app/assets/97010868/247bba41-9fee-4ddd-829c-b0e89310f0fb)
@NikitK-deriv we don't need to send event because its already handled in the core analytics package

Reply:
nope, it's already deprecated version. Please use deriv/analytics 1.3.0